pkg/commands/deps.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/mutate"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/google/ko/internal/sbom"
+	"github.com/sigstore/cosign/pkg/oci/signed"
```

Comment:
Is this `signed` package new? Why is there `oci.SignedImage` and `oci/signed.Image`?

Reply:
No, it is a convenient way of turning a `v1.Image` into an `oic.SignedImage` with empty signatures et al